docker/Dockerfile

```diff
@@ -158,13 +172,23 @@ RUN apt-get install -y \
 COPY --from=build /opt/conda /opt/conda
 ENV PATH=/opt/conda/bin:${PATH}
 
-# permit mpirun as root for Azure cluster submissions
-ENV OMPI_ALLOW_RUN_AS_ROOT=1
-ENV OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
+# make ssh/sshd less strict for wiring containers on Azure VM scale set
+# note: use 'service ssh start' to launch sshd (will fail if 22 in use)
+# note: can also set port != 22 and set port=X in MPI hosts file
+# note: need to setup password free ssh login between MPI hosts
+RUN sed -i 's/#PermitRootLogin prohibit-password/PermitRootLogin yes/g' \
+        /etc/ssh/sshd_config &&\
+    sed -i 's/#StrictModes yes/StrictModes no/g' \
+        /etc/ssh/sshd_config &&\
+    sed -i 's/#   StrictHostKeyChecking ask/    StrictHostKeyChecking no/g' \
+        /etc/ssh/ssh_config &&\
+    mkdir /run/sshd &&\
+    chmod u=rwx,go=rx /run/sshd
 
 # clean\finalize environment
 RUN conda remove -y cmake ninja &&\
     apt-get remove -y build-essential &&\
+    apt-get install -y vim &&\
```

Comment:
Was this added for debugging only? Should it be in the final image?

Reply:
Aw, yes. This was while debugging.. Let me remove.